tests/php/bootstrap.php

```diff
@@ -18,7 +18,16 @@
 	exit( 'Could not determine test root directory. Aborting.' );
 }
 
-$wp_tests_dir = getenv( 'WP_TESTS_DIR' ) ? getenv( 'WP_TESTS_DIR' ) : '/tmp/wordpress-tests-lib';
+$wp_tests_dir = getenv( 'WP_TESTS_DIR' );
+
+if ( ! $wp_tests_dir ) {
+	$wp_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
```

Comment:
This is generated by the `wp` CLI. It also fixed the problem on my system where my "temp" folder isn't just `tmp/`, it was like `/private/var/folders/blah/blahblhablhablah/T/blahblahblah....`